test/action.yaml

```
@@ -25,26 +25,36 @@ runs:
     - name: Checkout
       uses: actions/checkout@v3
       if: inputs.checkout-repo
+    - name: Verify needed scripts exist
+      id: verify-scripts-exist
+      shell: bash
+      run: |
+        if [[ -f ./script/setup && -f ./script/test ]]; then
```

Probably should use `-x` here to make sure it can be run later in the CI.
```suggestion
        if [[ -x ./script/setup && -x ./script/test ]]; then
```

---

Good point.